src/goban/tests.rs

```diff
@@ -1,7 +1,7 @@
 use crate::bitboard::BitBoard;
 use crate::goban::Goban;
 
-use super::Fscore;
+use crate::goban::fscore::Fscore;
```

Comment:
Is there a reason why you preferred `crate` over `super` here?

Reply:
Not really but making things explicit might help people who aren't aquainted with rust